src/ComposedModal/ComposedModal.svelte

```diff
@@ -69,6 +69,9 @@
   });
 
   function focus(element) {
+    if(selectorPrimaryFocus === null) {
```

Comment:
```suggestion
    if(selectorPrimaryFocus == null) {
```

A loose equals would suffice here